package.json

```diff
@@ -5,7 +5,7 @@
   "main": "dist/lib/index.js",
   "typings": "dist/lib/index.d.ts",
   "scripts": {
-    "test": "mocha dist/tests",
+    "test": "git submodule update --init && mocha dist/tests",
```

Comment:
Maybe you can split the submodule work and the actual test running into separate scripts that both get called from `test` script?